src/Legacy/Storage.php

```diff
@@ -2795,8 +2795,8 @@ public function getUri($title, $id = 0, $contenttypeslug = "", $fulluri = true,
         $id = intval($id);
         $slug = $this->app['slugify']->slugify($title);
 
-        // Don't allow strictly numeric slugs.
-        if (is_numeric($slug)) {
+        // Don't allow strictly numeric slugs, unless allow_numeric_slugs options is set
+        if (is_numeric($slug) && $contenttype['allow_numeric_slugs'] != 'true') {
```

Comment:
Can you please not use loose comparison, and `true` should not be a string but a boolean, e.g.
```php
if (is_numeric($slug) && $contenttype['allow_numeric_slugs'] !== true) {
```